pkg/skaffold/runner/runner.go

```diff
@@ -139,15 +139,28 @@ func newTaggerForConfig(t v1alpha2.TagPolicy) (tag.Tagger, error) {
 	return nil, fmt.Errorf("Unknown tagger for strategy %s", t)
 }
 
-// Run runs the skaffold build and deploy pipeline.
-func (r *SkaffoldRunner) Run(ctx context.Context) error {
-	if r.opts.DevMode {
-		if r.opts.Cleanup {
-			return cleanUpOnCtrlC(ctx, r.dev, r.cleanup)
-		}
-		return r.dev(ctx)
+// Build builds the artifacts.
+func (r *SkaffoldRunner) Build(ctx context.Context) error {
+	bRes, err := r.build(ctx, r.config.Build.Artifacts)
+
+	for _, res := range bRes.Builds {
```

Comment:
This could live in r.build and be exposed there. Or if you dont think its appropriate for dev/run mode, we can add it in cmd/

Reply:
`Build` and `build` don't take the same parameters. I'd rather try to refactor that in a separate PR